integration_tests/integration_tests.feature

```diff
@@ -35,14 +35,12 @@ Feature: Run integration tests
      |        default_5       |    24    |      60     |      60     |
      |        default_csv     |    24    |      60     |      60     |
 
-  # TODO: Once iaa_fee is included in billing, this test to be run non-zero iaa_fee
   Scenario Outline: Test Balanced Energy Bills
     Given we have a scenario named <scenario>
     And d3a is installed
-    When we run the simulation with setup file <scenario> and parameters [24, 60, 60, 0, 1]
+    When we run the simulation with setup file <scenario> and parameters [24, 60, 60, 1]
     Then the traded energy report the correct accumulated traded energy
     And the energy bills report the correct accumulated traded energy price
-    And the energy bills report the correct external traded energy and price
```

Comment:
Umm, can you explain the reason to remove this step?

Reply:
Please have a look in to the step function , it is disabled anyway

Reply:
Readding it again because of the other discussion